cli/js/compiler.ts

```diff
@@ -291,6 +300,19 @@ async function processImports(
   assert(sourceFiles.length === specifiers.length);
   for (let i = 0; i < sourceFiles.length; i++) {
     const sourceFileJson = sourceFiles[i];
+    if (sourceFileJson.mediaType === MediaType.Wasm) {
+      util.log(
+        "compiler::processImports: WASM import",
+        sourceFileJson.filename
+      );
+      // Create declaration file on the fly.
+      const _ = new SourceFile({
```

Comment:
Why even assign this to a variable?  It is never read.

But good way of tackling the problem.  The interesting thing will be if we can actually get the shape of the named exports of the WASM file, and actually pass into the compiler a dynamically generated set of source code.